dask/values.yaml

```diff
@@ -131,6 +131,76 @@ worker:
+      default_resources:  # overwritten by resource limits if they exist
+        cpu: 1  # Default CPU (DEPRECATED use `resources`).
+        memory: "4GiB"  # Default memory (DEPRECATED use `resources`).
+      env:  # Environment variables. See `values.yaml` for example values.
+      #  - name: EXTRA_APT_PACKAGES
+      #    value: build-essential openssl
+      #  - name: EXTRA_CONDA_PACKAGES
+      #    value: numba xarray -c conda-forge
+      #  - name: EXTRA_PIP_PACKAGES
+      #    value: s3fs dask-ml prometheus-client --upgrade
+      extraArgs: [] # Extra CLI arguments to be passed to the worker
+        # - --preload
+        # - worker-setup.py
+      resources: {}  # Worker pod resources. See `values.yaml` for example values.
+      #  limits:
+      #    cpu: 1
+      #    memory: 3G
+      #    nvidia.com/gpu: 1
+      #  requests:
+      #    cpu: 1
+      #    memory: 3G
+      #    nvidia.com/gpu: 1
+      mounts: {} # Worker Pod volumes and volume mounts, mounts.volumes follows kuberentes api v1 Volumes spec. mounts.volumeMounts follows kubernetesapi v1 VolumeMount spec
+      #  volumes:
+      #    - name: data
+      #      emptyDir: {}
+      #  volumeMounts:
+      #    - name: data
+      #      mountPath: /data
+      annotations: {}  # Annotations
+      tolerations: []  # Tolerations.
+      affinity: {}  # Container affinity.
+      nodeSelector: {}  # Node Selector.
+      securityContext: {}  # Security Context.
+      # serviceAccountName: ""
+      # port: ""
+      portDashboard: 8790 # Worker dashboard and metrics port.
+      #  this option overrides "--nthreads" on workers, which defaults to resources.limits.cpu / default_resources.limits.cpu
+      #  use it if you need to limit the amount of threads used by multicore workers, or to make workers with non-whole-number cpu limits
+      # threads_per_worker: 1
+      metrics:
+        enabled: false # Enable workers metrics. Pip package [prometheus-client](https://pypi.org/project/prometheus-client/) should be present on workers.
+        podMonitor:
+          enabled: false # Enable workers podmonitor
+          namespace: "" # Deploy podmonitor in different namespace, e.g. monitoring.
+          namespaceSelector: {} # Selector to select which namespaces the Endpoints objects are discovered from.
+          # Default: scrape .Release.Namespace only
+          # To scrape all, use the following:
+          # namespaceSelector:
+          #   any: true
+          additionalLabels: {} # Additional labels to add to the PodMonitor metadata.
+          interval: 30s # Interval at which metrics should be scraped.
+          jobLabel: "" # The label to use to retrieve the job name from.
+          podTargetLabels: [] # PodTargetLabels transfers labels on the Kubernetes Pod onto the target.
+          metricRelabelings: [] # MetricRelabelConfigs to apply to samples before ingestion.
```

Comment:
There is another template for handling the metrics configuration.

https://github.com/dask/helm-chart/blob/main/dask/templates/dask-worker-podmonitor.yaml

I think we should just delete this here and add a comment to the `worker.metrics` config to note that it will apply to the additional worker groups too.